infrastructure/main.tf

```diff
@@ -32,4 +32,16 @@ module "case-management-web" {
     PRINT_SERVICE_URL_REMOTE = "${var.ccd_print_service_url}"
     WEBSITE_NODE_DEFAULT_VERSION = "8.9.4"
   }
+
+  module "ccd-case-management-web-vault" {
+    source              = "git@github.com:hmcts/moj-module-key-vault?ref=master"
+    name                = "ccd-case-management-web${var.env}" // Max 24 characters
+    product             = "${var.product}"
+    env                 = "${var.env}"
+    tenant_id           = "${var.tenant_id}"
+    object_id           = "${var.jenkins_AAD_objectId}"
```

Comment:
This needs to be added to `variables.tf` (see https://github.com/hmcts/ccd-data-store-api/blob/master/infrastructure/variables.tf#L32)